CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Changelog
+
+## Unreleased
+
+### Breaking Changes
+
+- None
+
+### Added
+
+- None
+
+### Fixed
+
+- None
+
+## 3.0.0 - 2020-06-10
+
+### Breaking Changes
+
+- Cucumber 3 support removed (#762)
+
+### Added
+
+- Cucumber 4 support (#762)
+
+### Fixed
+
+- Fix a bundler deprecation when running specs (#761)
+- remove name override logic that never worked (#758)
+
+### Dependencies
+
+- Drop ruby 2.3 support (#760)
+- Drop ruby 2.2 support (#759)
```

Comment:
maybe just list these under breaking ?

Reply:
Happy to move them if you want. In my own projects, I don't like to set the expectation that dependency updates will always mean major bumps. And, SemVer does not consider dependency updates to be breaking changes (as long as the public API stays the same).